src/Activation/ActivateProximityTracing.tsx

```diff
@@ -20,15 +22,24 @@ const ActivateProximityTracing: FunctionComponent = () => {
   const { t } = useTranslation()
   const navigation = useNavigation()
 
+  const { setOnboardingToComplete } = useOnboardingContext()
   const { exposureNotifications } = usePermissionsContext()
 
   const handleOnPressEnable = () => {
     exposureNotifications.request()
-    navigation.navigate(ActivationScreens.NotificationPermissions)
+    platformDependentNavigation()
   }
 
   const handleOnPressDontEnable = () => {
-    navigation.navigate(ActivationScreens.NotificationPermissions)
+    platformDependentNavigation()
+  }
+
+  const platformDependentNavigation = () => {
```

Comment:
What do you think about
```suggestion
  const navigateToNextScreen = () => {
```
?